tests/mm0_mmu/pass/whitespace2.mm0

```diff
@@ -0,0 +1,3 @@
+--    v-- this is a LINE FEED
+sort s
```

Comment:
This should be a fail, right?

Reply:
LINE FEED is '\n', right ?

Reply:
I assumed that because of this :
https://fr.wikipedia.org/wiki/Carriage_Return_Line_Feed
But The internet doesn't hold the truth. At my age, I should know better. :)

Reply:
This is a '\n' in the test, I should have called it a NEWLINE, instead of a LINE FEED ?

Reply:
so it is. Wikipedia seems to prefer the name [newline](https://en.wikipedia.org/wiki/Newline), and so do I.